config/sample_application.yml

```diff
@@ -15,4 +15,4 @@ PUSHER_CLUSTER:
 
 TWITCH_CLIENT_ID:
 TWITCH_CLIENT_SECRET:
-TWITCH_WEBHOOK_SECRET:
+TWITCH_WEBHOOK_SECRET:
```

Comment:
missing new line :D

Reply:
This file is killing me. Meant to save my details for Gitpod and somehow I've included that in my PR. Haha.

Thank you! :-)